aiida/cmdline/params/types/identifier.py

```diff
@@ -10,7 +10,7 @@
 """
 Module for custom click param type identifier
 """
-from abc import ABC, abstractproperty
+from abc import ABC, abstractmethod
```

Comment:
what is wrong with abstractproperty?

Reply:
It's been deprecated since python 3.3: https://docs.python.org/3/library/abc.html?highlight=abstractproperty#abc.abstractproperty

Reply:
Huh, you don't say. I wonder why that warning never showed up? I remember the deprecation warnings from the `collections` module showing up, so I don't think that by default deprecation messages from built-in modules don't show. Anyway, good to keep these changes then :+1: